Node.js/basicrequest.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018, 2019 IBM Corp.
+ * Copyright 2018, 2019, 2022 IBM Corp.
```

Comment:
replace with 

Copyright 2018, 2022 IBM Corp.


pattern is to use initial and latest dates only. The implication is the full range.